tests/pri_kb_test.h

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef MU_PRI_KB_TEST_H
+#define MU_PRI_KB_TEST_H
+
+int run_pri_kb_tests(void);
+
+#endif // MU_PRI_KB_TEST_H
```

Comment:
>MU_PRI_KB_TEST_H [](http://example.com/codeflow?start=10&length=16)

Technically this would be PRI_KB_TEST_H to match the filename.

Reply:
changed.